### PR TITLE
Improve touch buttons

### PR DESCRIPTION
**Pull request by GTcreyon**
# Description of changes
Implements @Charpurrr's improved touch button sprites.

### CODE REVIEW

**Review by dignityloss — COMMENTED** [review]
touch_button.gd line 26

why is pos edited directly after declaration? couldn't it just be 
var pos = (Vector2(floor(index / 7), index % 7)) * Vector2(40, 21)

**Comment by GTcreyon** [comment]
> touch_button.gd line 26
> 
> why is pos edited directly after declaration? couldn't it just be 
> var pos = (Vector2(floor(index / 7), index % 7)) * Vector2(40, 21)

I found it more legible when split onto two lines.

File: classes/global/touch_control/touch_button.gd [hunk 1]
```diff
@@ -12,9 +22,11 @@ func _ready():
 
 
 func _setup_textures(new_id: String):
-	var ypos = ID_LIST.find(new_id) * 21
-	normal.region.position.y = ypos
-	pressed.region.position.y = ypos
+	var index = ID_LIST.find(new_id)
+	var pos = Vector2(floor(index / 7), index % 7)
+	pos *= Vector2(40, 21)
```
Comment (Koopa1018) [inline classes/global/touch_control/touch_button.gd:27]: Some comments explaining this math would be very appreciated

Comment (jaschutte) [reply classes/global/touch_control/touch_button.gd:27]: +could the 7 be moved to a variable rather than sitting there as a magic number?

Comment (GTcreyon) [reply classes/global/touch_control/touch_button.gd:27]: I must admit I rushed this one a bit. Will fix soon.

**Pull request merged**